app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+        Person.people[name] = self
+
+
+def create_person_list(people: dict) -> list:
+    person_list = []
+    for person_data in people:
+        Person(person_data["name"], person_data["age"])
+
+    for char in people:
```

Comment:
char is 1 letter, I guess it should've been character, if you wanted it that way, leave it as `character` )